src/routes/routes.ts

```diff
@@ -0,0 +1,6 @@
+import createPath from './createPath';
+
+export const getStarted = createPath(
+  '/docs/get-started',
+  () => `/docs/get-started`,
+);
```

Comment:
The original intention for creating this file was so that it could be used from our Markdown files. 
So instead of 

```mdx
// getting-started.mdx

## Developer Portal

The [Developer Portal](https://developers.raydiant.com) is where you can create and manage all your Raydiant apps as well as create API keys for our On-Brand Menu API.
```
We'd have

```mdx
import routes from '../../routes';

## Developer Portal

The [Developer Portal](routes.developerPortalUrl) is where you can create and manage all your Raydiant apps as well as create API keys for our On-Brand Menu API.
```

I'm mindful of not overcomplicating creating/editing documentation process - especially for non-technical folks. What are your thoughts? Should we reference this helper from Markdown files? This is currently only being used in the footer for the moment.

Reply:
Hm, don't have a strong opinion here but I am inclined to keep it simple for the main content. I think doing this in the footer is fine though as I don't see others needing to change it.